aip/general/0127.md

```diff
@@ -78,8 +78,10 @@ message CreateBookRequest {
   - URIs **may** use nested fields for their variable names. (Additionally,
     AIP-134 mandates this for `Update` requests.)
   - URIs **must** use the `*` character to represent ID components, which
-    matches all URI-safe characters except for `/`. URIs **may** use `**` as
-    the final segment of a URI if matching `/` is required.
+    matches all URI-safe characters except for `/`.
+  - URIs **must not** use `**`, except in the final segment of the URI.
+  - URIs **should not** use `**` as the final segment of a URI unless doing so
```

Comment:
Why not **must not**?

Reply:
In conversations with @jskeet he suggested not being that strict on that guidance, in case there APIs that intentionally want to model themselves after things like a filesystem.

Personally I'm not sure the benefit of expressing yourself as a filesystem is worth the lack of uniformity in path semantics, but *should not* was the middle ground.